pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/PersistentTopics.java

```diff
@@ -173,6 +173,34 @@ public void createPartitionedTopic(
         }
     }
 
+    @PUT
+    @Path("/{property}/{cluster}/{namespace}/{topic}/partitions/properties")
+    @ApiOperation(hidden = true, value = "Create a partitioned topic.",
```

Comment:
I think we don't need to add new features in v1 version.

Reply:
Ok, I will remove it.